From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@rmaclean__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

mimig1 commented:
Thanks for the contribution rmaclean! I actually attempted to fix this a few days ago, then learned it's a known issue with the site caused by the NOTE. To get this fixed in the short term, I'll file a PR to remove the NOTE and put that text in #5. (Ross, you can accept this PR.)

itechedit committed:
edit pass: active-directory-b2c-user-migration

itechedit committed:
edit pass: active-directory-b2c-user-migration

itechedit committed:
Update active-directory-b2c-user-migration.md

ShawnJackson committed:
edit pass: storage-files-snapshots

v-vijanu committed:
Merge pull request #1 from MicrosoftDocs/master

sync with master

v-vijanu committed:
Bug - 1087532 : Service Fabric CLI link added.

v-vijanu committed:
Bug - 1087532 - link updated

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into storage-files-snapshots

mjrousos committed:
Update links in SF programmatic scaling docs

mjrousos committed:
A few small style updates based on Acrolinx feedback

ShawnJackson committed:
edit pass: storage-files-snapshots

ShawnJackson committed:
edit pass: storage-files-snapshots

ShawnJackson committed:
Update storage-how-to-use-files-snapshots.md

ShawnJackson committed:
Update storage-how-to-use-files-snapshots.md

ShawnJackson committed:
Update storage-snapshots-files.md

Commit:
Update point-to-site-vpn-client-configuration-azure-cert.md

Made the following changes
1. Changed the title of the section that describes how to create VPN client configuration on Macs and re-worded the intro para
2. Modified the description of step #6 for Mac VPN configuration.

johnkemnetz committed:
Update monitoring-activity-log-schema.md

ShawnJackson committed:
Merge branch 'master' into storage-files-snapshots

Commit:
service-fabric-tutorial-create-cluster-azure-ps

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into service-fabric-tutorial-create-cluster-azure-ps2

Commit:
service-fabric-tutorial-create-cluster-azure-ps

billmath committed:
updating toc

itechedit committed:
Update active-directory-b2c-user-migration.md

mmacy committed:
[acr] CLI quickstart managed registry update

cherylmc committed:
graphics

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into creategw

cherylmc committed:
update

cephalin committed:
fix broken link

cephalin committed:
fix missing docker repo

cephalin committed:
acrolinx

cephalin committed:
revert files

cephalin committed:
acrolinx

ktoliver committed:
edit pass: enable-profiler-compute

ktoliver committed:
more edits

v-thepet committed:
fix validation warning

LoriMerrick committed:
Changes made to some of the heading levels.

v-thepet committed:
fix conflict

v-thepet committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fixy

cherylmc committed:
graphics

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into creategw

pharring committed:
Expand on ASP.NET Core Snapshot Collector configuration

seanknox committed:
AKS: add step to enable AKS feature on a subscription

cephalin committed:
merge conflicts

cephalin committed:
private registry update

cephalin committed:
prmerger feedback

kgremban committed:
IoT freshness

Lisaco88 committed:
Merge pull request #26180 from cephalin/docker-merge

Docker merge

billmath committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into app1

ShawnJackson committed:
Update active-directory-b2c-user-migration.md

rothja committed:
Fixing firewall reload command

PRMerger18 committed:
Merge pull request #25736 from aanandr/patch-2

Update point-to-site-vpn-client-configuration-azure-cert.md

neilpeterson committed:
updated node count

ShawnJackson committed:
Update enable-profiler-compute.md

ShawnJackson committed:
Merge pull request #25531 from itechedit/active-directory-b2c-user-migration-2

edit pass: active-directory-b2c-user-migration

v-thepet committed:
fix links

ShawnJackson committed:
Merge pull request #26114 from ktoliver/enable-profiler-compute

edit pass: enable-profiler-compute

PRMerger16 committed:
Merge pull request #26208 from neilpeterson/aks-node-count

updated node count

Commit:
Merge pull request #26209 from v-thepet/newfix

production bug: fix links

v-shils committed:
Merge pull request #26043 from cherylmc/creategw

Creategw

Ja-Dunn committed:
Merge pull request #25653 from ShawnJackson/storage-files-snapshots

edit pass: share snapshot articles

PRMerger16 committed:
Merge pull request #26173 from pharring/master

Expand on ASP.NET Core Snapshot Collector configuration

Commit:
Merge pull request #26207 from rothja/patch-23

Fixing firewall reload command

davidmu1 committed:
cleaned up TOC

dominicbetts committed:
Add examples

PRMerger18 committed:
Merge pull request #26215 from dominicbetts/suite-cf-commands

Add examples

billmath committed:
fixing spacing

Commit:
address PR feedback

seanmck committed:
Make location in response JSON match request

PRMerger16 committed:
Merge pull request #26177 from seanknox/aks-quickstart-update

AKS: add step to enable AKS feature on a subscription

Commit:
Merge pull request #25627 from mjrousos/patch-3

Update links in SF programmatic scaling docs

v-thepet committed:
fix merge conflict

Commit:
Merge pull request #26219 from seanmck/aks-quickstart-regionfix

Make location in response JSON match request

johnkemnetz committed:
Update monitoring-activity-log-schema.md

Ja-Dunn committed:
Merge pull request #26152 from v-thepet/fixy

fix validation warning: redirect active-directory/active-directory-preview-explainer

itechedit committed:
edit pass: security-center-azure-policy

dsk-2015 committed:
X.509 blurb added to the IoT Hub Security CA certs

cynthn committed:
Update virtual-machines-faq-for-disks.md

Ja-Dunn committed:
Merge pull request #25990 from billmath/app1

updating toc

davidmu1 committed:
cleaned up TOC

Ja-Dunn committed:
Merge pull request #25565 from v-vijanu/master

Bug - 1087532 : Service Fabric CLI link added.

Commit:
Merge pull request #26233 from cynthn/master

Update virtual-machines-faq-for-disks.md

PRMerger16 committed:
Merge pull request #26004 from mmacy/acr-cli-update

[acr] CLI quickstart managed registry update

Ja-Dunn committed:
Merge pull request #26230 from davidmu1/appgwtoc

cleaned up TOC

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

markgalioto committed:
fixing an Azure cmd that was turned into an icon

JasonWHowell committed:
Update howto-configure-ssl.md

ShawnJackson committed:
Update service-fabric-tutorial-create-cluster-azure-ps.md

PRMerger17 committed:
Merge pull request #25781 from johnkemnetz/patch-1

Update monitoring-activity-log-schema.md

PRMerger17 committed:
Merge pull request #26236 from dsk-2015/dsk_iotf17_b2

X.509 bash scripts reference added to the IoT Hub Security CA certs

v-alje committed:
Merge pull request #26238 from MicrosoftDocs/FromPublicRepo

From public repo

Ja-Dunn committed:
Merge pull request #26211 from kgremban/oct24_iotfreshness

IoT freshness

Commit:
Merge pull request #26239 from markgalioto/ab-10242017

fixing an Azure cmd that was turned into an icon

ShawnJackson committed:
Update security-center-azure-policy.md

Commit:
Merge pull request #26240 from JasonWHowell/patch-1

MySQL: Update howto-configure-ssl.md

ShawnJackson committed:
Merge pull request #25894 from LoriMerrick/service-fabric-tutorial-create-cluster-azure-ps2

edit pass: service-fabric-tutorial-create-cluster-azure-ps-2

ShawnJackson committed:
Merge pull request #26232 from itechedit/security-center-azure-policy-2

edit pass: security-center-azure-policy
